staging/src/k8s.io/apiserver/pkg/util/flag/map_string_bool_test.go

```diff
@@ -36,7 +34,9 @@ func TestStringMapStringBool(t *testing.T) {
 	for _, c := range cases {
 		t.Run(c.desc, func(t *testing.T) {
 			str := c.m.String()
-			assert.Equal(t, c.expect, str)
+			if c.expect != str {
```

Comment:
is there a particular reason for moving from using assert/require to reflect?

Reply:
Yeah, it turns out that testing assert helpers are against the style guide.

Reply:
good to know :)